assets/js/googlesitekit/modules/index.js

```diff
@@ -30,12 +30,54 @@ import './datastore';
 
 const Modules = {
 	createModuleStore,
+	/**
+	 * Activates a module on the server.
+	 *
+	 * Activate a module (based on the slug provided).
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {any}    args      Module action arguments.
+	 * @param {string} args.slug Slug of the module to activate.
+	 * @return {Promise} The dispatched action.
+	 */
 	activateModule: ( ...args ) => {
```

Comment:
See above comment, let's change this to `( slug )` and use the same documentation as for the store action parameter.